### PR TITLE
MSTR-202: Add onebill API definitions

### PR DESCRIPTION
**Pull request by masmerino13**

### CODE REVIEW

File: src/js/lib/jquery.kazoosdk.js [hunk 1]
```diff
@@ -558,6 +558,10 @@
 				'deleteNotification': { verb: 'DELETE', url: 'accounts/{accountId}/notifications/{notificationId}' },
 				'getDnsEntries': { verb: 'GET', url: 'accounts/{accountId}/whitelabel/domains' },
 				'checkDnsEntries': { verb: 'POST', url: 'accounts/{accountId}/whitelabel/domains?domain={domain}' }
+			},
+			onebillIntegration: {
+				'get': { verb: 'GET', url: 'accounts/{accountId}/cdr_push_onebill' },
```
Comment (azefiel) [inline src/js/lib/jquery.kazoosdk.js:563]: What does this resource is expected to return, a single document or list of documents?

File: src/js/lib/jquery.kazoosdk.js [hunk 2]
```diff
@@ -558,6 +558,10 @@
 				'deleteNotification': { verb: 'DELETE', url: 'accounts/{accountId}/notifications/{notificationId}' },
 				'getDnsEntries': { verb: 'GET', url: 'accounts/{accountId}/whitelabel/domains' },
 				'checkDnsEntries': { verb: 'POST', url: 'accounts/{accountId}/whitelabel/domains?domain={domain}' }
+			},
+			onebillIntegration: {
```
Comment (azefiel) [inline src/js/lib/jquery.kazoosdk.js:562]: Method group should match endpoint name for easy "guessing".
```suggestion
			cdrPushOnebill: {
```

Also, please try to keep this map alphabetically sorted, to allow for fast visual lookup.

File: src/js/lib/jquery.kazoosdk.js (hunk 1 above)
Comment (masmerino13) [reply src/js/lib/jquery.kazoosdk.js:563]: This will return a single document

**Review by azefiel — APPROVED** [review]
Looks good, merge away!

**Pull request merged**